packages/shared/src/open-feature.ts

```diff
@@ -124,7 +124,15 @@ export abstract class OpenFeatureCommonAPI<P extends CommonProvider = CommonProv
 
     const emitters = this.getAssociatedEventEmitters(clientName);
 
-    if (typeof provider.initialize === 'function') {
+    // TODO: this can be removed after 1.0
```

Comment:
Is there any harm in keeping this in post 1.0?

Reply:
I guess not really, except for that it's a bit messy.

Reply:
I've removed this comment. I think that since people could be using pure JS, it's nice to keep warnings like this in the SDK. That something I tend to forget.